frontend/src/components/page-about/index/leadership.module.css

```diff
@@ -0,0 +1,41 @@
+@config "../../../config/tailwind.config.js";
+
+.title-container {
+  @apply text-center pb-11.25;
+}
+
+.container {
+  @apply pb-20.25;
+  display: flex;
```

Comment:
I opted to use plain css instead of tailwind for stuff that I thought was specific to this page

Reply:
you can do this in tailwind.  flex = display: flex, flex-col = flex-direction: column, items-center = align-items: center  tailwind could handle all the normal css which we do not need write.  plus if the is apply to specific break points for response web design we just at md: or lg:  or sm: to apply at the specific breakpoint manage by tailwind.  each tag/class name reference a single property / value.  which that is great flexibility of tailwind for no need to use plain css.

Reply:
Nvm I got it to work!

Reply:
From their docs it looks like they typically write the classes inline and only use a separate CSS file if it's a complex component

https://tailwindcss.com/docs/reusing-styles